x/emissions/keeper/keeper.go

```diff
@@ -340,14 +342,20 @@ func (k *Keeper) IsReputerNonceUnfulfilled(ctx context.Context, topicId TopicId,
 	if err != nil {
 		return false, err
 	}
-
+	if nonce == nil {
+		return false, errors.New("nil reputer nonce provided")
+	}
 	// Check if the nonce is present in the unfulfilled nonces
 	for _, n := range unfulfilledNonces.Nonces {
+		if n == nil {
```

Comment:
necessary?

Reply:
in theory not needed, but i wanted to make sure we see this if it happens